README.md

```diff
@@ -1,57 +1,136 @@
-# Project Name
+---
+page_type: sample
+languages:
+- javascript
+- typescript
+products:
+- angular
+- react
+- nodejs
+- ms-graph
+- microsoft-identity-platform
+- azure-active-directory
+- azure-active-directory-b2c
+- azure-app-services
+- azure-storage
+- azure-key-vault
+description: "Microsoft identity platform & Microsoft Authentication Library for JavaScript Tutorial"
+urlFragment: "ms-identity-javascript-tutorial"
+---
 
-(short, 1-3 sentenced, description of the project)
+# Microsoft identity platform & Microsoft Authentication Library for JavaScript Tutorial
 
-## Features
+[Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/v2-overview), along with [Azure Active Directory Azure AD](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-whatis) and [Azure Azure Active Directory B2C (Azure AD B2C)](https://docs.microsoft.com/azure/active-directory-b2c/overview) are central to **Azure** cloud ecosystem. This tutorial aims to take you through the fundamentals of modern authentication with JavaScript, using the [Microsoft Authentication Library for JavaScript (MSAL.js)](https://github.com/AzureAD/microsoft-authentication-library-for-js).
 
-This project framework provides the following features:
+We recommend following the chapters in successive fashion, in your platform of choice (**Vanilla JavaScript**, **Angular**, **React**, **Node**) but code samples are self-contained so feel free to pick samples by topics that you may need at the moment.
 
-* Feature 1
-* Feature 2
-* ...
+> :warning: This is a *work in progress*. Come back frequently to discover more samples.
 
-## Getting Started
+## Prerequisites
 
-### Prerequisites
+- [Node.js](https://nodejs.org/en/download/).
+- [Visual Studio Code](https://code.visualstudio.com/download).
+- [VS Code Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack).
```

Comment:
```suggestion
- [Node.js](https://nodejs.org/en/download/)
- [Visual Studio Code](https://code.visualstudio.com/download)
- [VS Code Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
```

This one is debatable. You typically either terminate all items in a bulleted list with periods, or none of them - the key is consistency. However, you have full sentences only in the last bullet, so really, this is an executive decision on styling that you can decline to take. 😉